atd-vzd/metadata/databases/default/functions/functions.yaml

```diff
@@ -1,8 +1,8 @@
 - "!include public_find_cr3_collisions_for_location.yaml"
-- "!include public_find_cr3_mainlane_crash.yaml"
 - "!include public_find_crash_in_jurisdiction.yaml"
 - "!include public_find_crash_jurisdictions.yaml"
 - "!include public_find_crashes_in_jurisdiction.yaml"
+- "!include public_find_location_for_noncr3_collision.yaml"
 - "!include public_find_location_for_cr3_collision.yaml"
```

Comment:
I ran into an inconsistent metadata warning when I applied the Hasura metadata locally. The first line added back here is a function that was dropped in https://github.com/cityofaustin/atd-vz-data/pull/1286 and the second one is being dropped in this PR. These two lines can go since Hasura doesn't need to worry about tracking or not tracking them anymore.